app/models/maestrano/connector/rails/concerns/entity.rb

```diff
@@ -231,42 +231,53 @@ def push_entities_to_connec_to(connec_client, mapped_external_entities_with_idma
     return unless self.class.can_write_connec?
 
     Maestrano::Connector::Rails::ConnectorLogger.log('info', organization, "Sending #{Maestrano::Connector::Rails::External.external_name} #{self.class.external_entity_name.pluralize} to Connec! #{connec_entity_name.pluralize}")
-    mapped_external_entities_with_idmaps.each do |mapped_external_entity_with_idmap|
-      external_entity = mapped_external_entity_with_idmap[:entity]
-      idmap = mapped_external_entity_with_idmap[:idmap]
-
-      begin
+    
+    request_per_call = 100
```

Comment:
because ideally it should be an opts with a default value to 100 but this methods does not has an opts parameter